src/bootlint.js

```diff
@@ -1188,6 +1191,9 @@ var LocationIndex = _location.LocationIndex;
                         seenLint = true;
                     }
 
+                    // Append the wiki URL to the lint message.
+                    lint.message += " (" + lint.url + ")";
```

Comment:
Instead of modifying the `message` property, use `lint.url` in the `console.warn()` calls.